src/js/common/components/authentication/LoginPrompt.jsx

```diff
@@ -9,20 +9,27 @@ class LoginPrompt extends React.Component {
   }
 
   componentDidMount() {
-    // event for google analytics that we prompted for login
     window.dataLayer.push({ event: 'login-prompt-displayed' });
   }
 
   handleLogin() {
+    window.dataLayer.push({ event: 'login-link-clicked' });
     const myLoginUrl = this.props.loginUrl;
-    const receiver = window.open(`${myLoginUrl}&op=signin`, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
-    receiver.focus();
+    if (myLoginUrl) {
```

Comment:
I think we still need it because the url could be null, e.g. if the component renders before the URL is returned. If it's null, we don't want to open up an empty window.

I am going to open a separate issue for deciding what to do if the request fails (e.g., if the user loses connectivity).